packages/core/src/runtime/compileCSS.ts

```diff
@@ -6,7 +6,7 @@ import { normalizeNestedProperty } from './utils/normalizeNestedProperty';
 export interface CompileCSSOptions {
   className: string;
 
-  pseudo: string;
+  selectors: string[];
```

Comment:
would the length of selectors ever exceed 2 ? It seems that the only case we are handling here is pseudo selector + element. It shouldn't be possible to go further since it would be invalid css ?

Reply:
Can you please provide an example?